app.js

```diff
@@ -6,12 +6,139 @@ const KNXInterfaceManager = require('./lib/KNXInterfaceManager');
 class KNXApp extends Homey.App {
 
   async onInit() {
+
     this.log('KNXApp is running...');
     const address = await this.homey.cloud.getLocalAddress();
     const homeyIP = address.split(':', 1).toString();
 
     this.log('Homey IP + Parsed IP', address, homeyIP);
     this.knxInterfaceManager = new KNXInterfaceManager(homeyIP);
+
+
+    const sendTelegramAction = this.homey.flow.getActionCard('knx_send');
+    sendTelegramAction.registerRunListener(this.sendKNXTelegram.bind(this));
+
+
+    this.receiveTelegramTrigger = this.homey.flow.getTriggerCard('knx_receive');
+    this.eventListenerGroupAddresses = [];
+
+    this.receiveTelegramTrigger.registerRunListener(async (args, state) => {
+      console.log("registerRunListener", args, state);
```

Comment:
Never reaches this line

Reply:
It does reach that line, but because we are inside a few async functions without any error handling the error that is actually thrown (cannot access trigger on undefined object) never surfaces. So should anyone every read this comment. 

In a situation where it seem a line is just never triggered, please remember that maybe it just throws an unhandled and unsurfaced (hidden) error.